app/templates/components/forms/events/view/create-access-code.hbs

```diff
@@ -0,0 +1,83 @@
+<form class="ui form" {{action 'submit' on='submit'}}>
+  <div class="field">
+    <label class="required">{{t 'Access Code'}}</label>
+    {{input type='text' name='access_code' value=data.code}}
+  </div>
+  <div class="field">
+    <label class="required">{{t 'Number of Access Tickets'}}</label>
+    {{input type='number' name='number_of_access_tickets' placeholder='0' min=0 value=ticketsNumber}}
+  </div>
+  <div class="grouped inline fields">
+    <label class="required">{{t 'Status'}}</label>
+    <div class="field">
+      {{ui-radio current='active' name='status' label='Active' value='active'}}
+    </div>
+    <div class="field">
+      {{ui-radio name='status' label='Inactive' value='inactive'}}
+    </div>
+  </div>
+  <div class="inline field">
+    <label class="required">{{t 'Select Tickets applied to the access code'}}</label>
+    <div class="ui hidden divider"></div>
+    {{ui-checkbox label='Select all Ticket types' name='all_ticket_types' checked=allTicketTypesChecked onChange=(action (mut allTicketTypesChecked))}}
+  </div>
+  {{#if allTicketTypesChecked}}
+    <div class="ui list">
+      {{#each data.tickets as |ticketName|}}
+        <div class="item">{{ticketName}}</div>
+      {{/each}}
+    </div>
+  {{/if}}
+  <div>
+    <div class="inline field">
+      {{ui-checkbox class='toggle' label=(t 'Show more options') checked=showMoreOptions onChange=(action (mut showMoreOptions))}}
+    </div>
+    {{#if showMoreOptions}}
+      {{t 'Set the min and max quantity allowed to purchase per order(Optional)'}}
+      <div class="{{unless device.isMobile 'fields'}}">
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Min'}}</label>
+          {{input type='number' name='min' min=0 value=data.minQuantity}}
+        </div>
+        {{#if device.isMobile}}
+          <div class="ui hidden divider"></div>
+        {{/if}}
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Max'}}</label>
+          {{input type='number' name='max' min=0 value=data.maxQuantity}}
+        </div>
+      </div>
+      <div class="ui hidden divider"></div>
+      {{t 'Validity Period(Optional)'}}
+      <div class="fields">
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Valid from'}}</label>
+          {{widgets/forms/date-picker id='start_date' value=data.validFromDate}}
+          <div class="ui hidden divider"></div>
+          {{widgets/forms/time-picker id='start_time' value=data.validFromTime}}
+        </div>
+        <div class="wide field {{if device.isMobile 'sixteen' 'five'}}">
+          <label>{{t 'Expires on'}}</label>
+          {{widgets/forms/date-picker id='end_date' value=data.validTillDate}}
+          <div class="ui hidden divider"></div>
+          {{widgets/forms/time-picker id='end_time' value=data.validTillTime}}
```

Comment:
@sumedh123 I think date and time picker needs validation since the `expires on` date should always be after the `valid from` date.